pytest_fixtures/component/katello_agent.py

```diff
@@ -1,57 +0,0 @@
-from box import Box
-import pytest
-
-from robottelo.config import settings
-from robottelo.utils.installer import InstallerCommand
-
-
-@pytest.fixture(scope='module')
-def sat_with_katello_agent(module_target_sat):
```

Comment:
Can we remove this too? https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/component/katello_agent.py

Reply:
The whole `pytest_fixtures/component/katello_agent.py` is being removed.